pkg/osquery/table/platform_tables_linux.go

```diff
@@ -17,5 +17,6 @@ func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, c
 		dataflattentable.TablePluginExec(client, logger,
 			"kolide_nmcli_wifi", dataflattentable.KeyValueType, []string{"/usr/bin/nmcli", "--mode=multiline", "--fields=all", "device", "wifi", "list"},
 			dataflattentable.WithKVSeparator(":")),
+		dataflattentable.TablePluginExec(client, logger, "kolide_lsblk", dataflattentable.JsonType, []string{"/usr/bin/lsblk", "-J"}),
```

Comment:
On a test device of mine (a DO server), `lsblk` is in my path but located at `/bin/lsblk`

Reply:
I can't remember - is there an easy way to specify multiple potential locations for the binary?

Reply:
Oh, it's great to know that. I don't think, but we can write one.

Reply:
Okay, I made one. See if it works on your DO machine. 

Any idea what the path to zfs is on that one?